cluster.go

```diff
@@ -225,7 +220,15 @@ func (c *clusterClient) _refresh() (err error) {
 	for master, g := range groups {
 		conns[master] = connrole{conn: c.connFn(master, c.opt), replica: false}
 		for _, addr := range g.nodes[1:] {
-			conns[addr] = connrole{conn: c.connFn(addr, c.opt), replica: true}
+			var cc conn
+			if c.opt.ReplicaOnly {
+				opt := *c.opt
```

Comment:
I am still a little confused. So now, are we requiring users to set `ReplicaOnly` as well even if they use `SendToReplicas`?

I thought we would only set `ReplicaOnly` transparently for users here to the `opt`.

Reply:
fix it. [3ec3548](https://github.com/redis/rueidis/pull/430/commits/3ec3548e8207c497029f37acad869db5e0d3e5f5) 

when `SendToReplicas` is configured and `ReplicaOnly` is true,
- connection to master: do not send "READONLY"
- connection to replica: send "READONLY"

when `SendToReplicas` is configured and `ReplicaOnly` is false,
- connection to master: do not send "READONLY"
- connection to replica: do not send "READONLY"

when `SendToReplicas` is not configured and `ReplicaOnly` is true,
- connection to master: send "READONLY"

Reply:
Why we don't send "READONLY" to replicas when SendToReplicas is configured and ReplicaOnly is false?

Isn't it the original issue you want to fix?

Reply:
[cd6a674](https://github.com/redis/rueidis/pull/430/commits/cd6a674ec54ce392de2ff6e0d268fc6cb877e344)

Yes, it doesn't matter HOW. i just want to send "READONLY" command to replica connections automatically. but if you think read-only mode as a default is better than i changed it.